tests/Unit/NumericalAlgorithms/Spectral/Test_ComplexDataView.cpp

```diff
@@ -163,6 +159,9 @@ void test_view() noexcept {
       make_not_null(&assign_view_source), assign_view_source.size()};
```

Comment:
the clang-tidy comment on line 154: I'm not sure where the corresponding `NOLINT` is

Reply:
When I changed the code in the last big change, the result for this block lost some pointer arithmetic it had before. I'll just remove the clang-tidy comment.